test/mnist-keras/extra-hosts.yaml

```diff
@@ -0,0 +1,5 @@
+version: '3.3'
+services:
+ client:
+  extra_hosts:
+    combiner-name: 13.53.197.49
```

Comment:
?

Reply:
What do you mean? Want a not-real IP instad in the example?

Reply:
ok, changed to non-funtional IPs in example.